app/assets/javascripts/3dbio_viewer/src/webapp/components/annotations-tool/AnnotationsTool.tsx

```diff
@@ -99,12 +101,25 @@ export const AnnotationsTool: React.FC<AnnotationsToolProps> = React.memo(props
 
     return (
         <Dialog open={true} onClose={onClose} maxWidth="lg">
-            <DialogTitle>
+            <StyledDialogTitle>
                 {i18n.t("Add annotation")}
+                <HtmlTooltip
```

Comment:
This pattern is duplicated in Network.tsx, maybe can we create a component `<DialogHelp><StyledTypography variant="body2">...</DialogHelp>`?